### PR TITLE
Create svloog.json

### PR DESCRIPTION
**Pull request by slvoog**

### CODE REVIEW

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/svloog.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "just gonna put some projects here idk thought this might be cool to add some personality",
+  "repo": "https://github.com/slvoog/slvoog.github.io",
+  "owner": {
+    "username": "slvoog",
+    "email": "svloogz@gmail.com",
+    "twitter": "svloogz"
+  },
+  "record": {
+    "CNAME": "slvoog.github.io"
```
[the remark this answers is not in the document]
Comment (slvoog) [reply domains/svloog.json:10]: Fixed

**Comment by slvoog** [comment]
Fixed the 404

**Review by MaskDuck — APPROVED** [review]
nice

**Comment by MaskDuck** [comment]
@slvoog go config github pages (you can find how in the docs) and optionally star the repo

**Comment by MaskDuck** [comment]
also feel free to join discord/open gh issue if youve ever need help

**Pull request merged**